container/rules/debian_pkg_tar.bzl

```diff
@@ -0,0 +1,185 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@io_bazel_rules_docker//container:container.bzl", _container_imge = "container")
+load("@base_images_docker//package_managers:download_pkgs.bzl", _download_deb_pkgs = "download")
+load("@base_images_docker//package_managers:apt_key.bzl", _apt_key = "key")
+
+debian_pkgs_attrs = _container_imge.image.attrs + _apt_key.attrs + _download_deb_pkgs.attrs + {
```

Comment:
lets not use _container_imge.image.attrs and just list out which specific attrs we want to allow

Reply:
You will also need a different list for _generate_deb_tar (which does not receive layers) vs _aggregate_debian_pkgs_impl (which should only receive layers).

Reply:
1. Will remove ```_container_imge.image.attrs``` and only add ```base``` attribute.
2. I think ```_aggregate_debian_pkgs_impl```  need to reuse this attribute list but add a new attribute ```language_layers```, we can restrict the use of other attributes (packages, additional_repos, etc) through input validation.